### ISSUE TITLE
Ability to disable Joystick actions?

### ISSUE THREAD

**Issue by Flexico** [MrCrayfish/Controllable#303]
I've been using a program called Antimicro to make my controller work in Minecraft, and I quite like the way it works! However, my only issue is that mapping the left joystick to WASD gives me no control over walking speed.

This is what I'd like to use this mod for -- ONLY to give the left stick control over walking speed. As far as I can tell, there is no way to enable/disable functionalities of the joysticks in the mod options; adding this would be epically helpful!

Thank for your time! ^_^

**Comment by MrCrayfish** [comment]
This mod already does everything you need without an intermediate program.

**Comment by Flexico** [comment]
It does not allow me to seamlessly tab out and interact with a browser or file window without touching a mouse or keyboard! With mouse movement/clicks and common buttons like Space and Shift, I can do loads beyond just games with my controller! The only thing it can't do (that I know of) is to apply varying speeds to WASD movement.

**Comment by MrCrayfish** [comment]
Sorry about my previous comment, I understand better now. I can add an option in the next update, it's just disabling the joystick right?

**Comment by Flexico** [comment]
Yeah, an enable/disable option for each joystick would be perfect! ^^ Thanks~

**Comment by Flexico** [comment]
Any progress on this? ^^

**Comment by Flexico** [comment]
Update...?

**Comment by MrCrayfish** [comment]
May I ask what modloader and mc version you want to use this on? I can quickly build a custom version of the mod that will fix the issue temporarily until I write a proper solution.

**Comment by Flexico** [comment]
Forge 1.18.2 and 1.19.4! ^^

**Comment by MrCrayfish** [comment]
It will now be possible to disable the joysticks (by unbinding them) in an upcoming update, that update will however only be on Minecraft 1.21.1 and above. Marking as closed as it's technically fixed.

Related commits:
https://github.com/MrCrayfish/Controllable/commit/6f43b7b1af9e75a00ccf6a01893afb64d31f7ebc
https://github.com/MrCrayfish/Controllable/commit/c8671ef7f2b38e5deeb12fa62e30dbd6308c60be
https://github.com/MrCrayfish/Controllable/commit/24519af0543d1cad72fc85bb5985aafbe4c96c83
